_shared/project/conf/alembic.ini

```diff
@@ -1,6 +1,6 @@
 [alembic]
-script_location = {{ cookiecutter.slug }}/migrations
-database_url: postgresql://postgres@localhost:{{ cookiecutter.__postgres_port }}/postgres
+script_location = {{ cookiecutter.package_name }}/migrations
+sqlalchemy.url = postgresql://postgres@localhost:4433/postgres
```

Comment:
`slug` (e.g. `test-pyramid-app`) is just wrong here, you want `package_name` (e.g. `test_pyramid_app`.

`database_url` doesn't work here: Alembic looks for `sqlalchemy.url`.